site/content/en/docs/manual/advanced/cli.md

```diff
@@ -46,13 +58,125 @@ optional arguments:
   --debug               show debug output
 ```
 
-**Examples**
+You can get help for each positional argument, e.g. `ls`:
+
+```bash
+cli.py ls -h
+```
+```
+usage: cli.py ls [-h] [--json]
+
+List all CVAT tasks in simple or JSON format.
+
+optional arguments:
+  -h, --help  show this help message and exit
+  --json      output JSON data
+```
+
+## Examples
+
+### Create
+
+Description of the options you can find in
+[Creating an annotation task](/docs/manual/basics/creating_an_annotation_task/) section.
+
+- Create a task named "new task" on the default server "localhost:8080", labels from the file "labels.json"
+  and local images "file1.jpg" and "file2.jpg", the task will be created as current user:
+  ```bash
+  cli.py create "new task" --labels labels.json local file1.jpg file2.jpg
```

Comment:
@TOsmanov , could you please add an example of labels.json? I cannot find its format in the tutorial. Also for some examples it will be better to specify labels directly in the command line.

Reply:
I added a example labels file, but it seems like entering labels as a string doesn't work.

Reply:
@TOsmanov , could you please submit a bug with corresponding steps to reproduce?

Reply:
@TOsmanov , labels should be in json format. Something like '[{'name': 'person'}]'

Reply:
> @TOsmanov , labels should be in json format. Something like `'[{"name": "person"}]'`

@nmanovic, it works, thanks